test/test_sign.py

```diff
@@ -24,3 +28,23 @@ def test_signer_production():
 def test_signer_staging():
     signer = Signer.staging()
     assert signer is not None
+
+
+@pytest.mark.online
+@pytest.mark.parametrize("signer", [Signer.production(), Signer.staging()])
+def test_sign_rekor_entry_consistent(signer):
+    token = detect_credential()
+    if token is None:
+        pytest.skip("no ambient credentials; skipping")
+
+    payload = secrets.token_bytes(32)
+    result = signer.sign(payload, token)
+
+    expected_entry = result.log_entry
+    actual_entry = signer._rekor.log.entries.get(log_index=expected_entry.log_index)
+
+    assert expected_entry.uuid == actual_entry.uuid
+    assert expected_entry.body == actual_entry.body
+    assert expected_entry.integrated_time == actual_entry.integrated_time
+    assert expected_entry.log_id == actual_entry.log_id
+    assert expected_entry.log_index == actual_entry.log_index
```

Comment:
```suggestion
    expected_entry = signer.sign(payload, token).log_entry
    actual_entry = signer._rekor.log.entries.get(log_index=expected_entry.log_index)

    assert expected_entry.uuid == actual_entry.uuid
    assert expected_entry.body == actual_entry.body
    assert expected_entry.integrated_time == actual_entry.integrated_time
    assert expected_entry.log_id == actual_entry.log_id
    assert expected_entry.log_index == actual_entry.log_index
```

Just a bit of simplification.